server/plugins/packages/themes.ts

```diff
@@ -82,10 +82,11 @@ function makePackageThemeObject(
 
 	const themeColor = /^#[0-9A-F]{6}$/i.test(module.themeColor) ? module.themeColor : null;
 	const modulePath = Config.getPackageModulePath(moduleName);
+
```

Comment:
try not to make unrelated changes that have nothing to do with your PR ;)

Reply:
Ahh my git was configured to hide only whitespace changes, I was confused as I was pretty sure I checked my diff and didn't saw something like that, now it won't EVER happen again!